src/DDTrace/Integrations/Laravel/V5/LaravelIntegrationLoader.php

```diff
@@ -52,8 +52,10 @@ public function load()
 
             $span = $self->rootScope->getSpan();
             try {
-                $user = auth()->user()->id;
-                $span->setTag('laravel.user', strlen($user) ? $user : '-');
+                $user = auth()->user();
+                if (null !== $user) {
```

Comment:
This is not required if starting from 5.0 we always have that same interface. Do you think it would be safer if is conditionally do this operation only if `is_a(... interface ...)`?

Reply:
Good point! Fixed. :)